src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -1174,7 +1174,9 @@ private StandaloneSignatureHandle GetBaselineLocalSignature(MethodDefinitionHand
             Debug.Assert(Thread.CurrentThread.GetApartmentState() == ApartmentState.MTA);
 
             var symMethod = (ISymUnmanagedMethod2)_pdbReader.Value.GetMethodByVersion(MetadataTokens.GetToken(methodHandle), methodVersion: 1);
-            return MetadataTokens.StandaloneSignatureHandle(symMethod.GetLocalSignatureToken());
+
+            // async methods do not have debug info.
```

Comment:
This is imprecise. Should rather be "async kick-off methods". Other methods might not have debug information either, so it would be better to say "Compiler generated methods (e.g. async kick-off methods) might not have debug information"